anonymizer/OpenDiffix.Service/Program.fs

```diff
@@ -54,7 +54,7 @@ let getAnonParams (requestAnonParams: RequestAnonParams) (salt: string) =
     Suppression = requestAnonParams.Suppression
     OutlierCount = requestAnonParams.OutlierCount
     TopCount = requestAnonParams.TopCount
-    NoiseSD = requestAnonParams.NoiseSD
+    NoiseSD = sqrt requestAnonParams.BaseNoiseSD
```

Comment:
Why is it passed as squared? It's technically a variance?

Reply:
embarrassing, should be `BaseNoiseSD / sqrt(2)`, I don't know why I did this, good catch!

Reply:
DfD uses total noise, while `reference` uses per-layer noise. Total noise scales with the square root of the number of noise samples. Currently, we have 2 noise layers. Meaning that `TotalSD = LayerSD * sqrt(2)` or `LayerSD = TotalSD / sqrt(2)`.

Reply:
ah, the reason is that `NoiseSD` is actually `noise_sd_layer`, which is a single layer SD. `base_sd` is for the sum of two noise layers, so the std dev is "split" between these layers.